### DISCUSSION TITLE
Change Angular Material table row background colour on hover

### DISCUSSION THREAD

**Discussion by caboodal** [tailwindlabs/tailwindcss#12304] [Help]
Hi,

I have an Angular material table and want to change the background colour on hover but cannot get this to work. My mat-row element is as shown below. I get the cursor changing to a pointer and if use the bg-blue-100 without the hover utility then I get all rows in blue but it wont change the row to blue when using the hover utility. This seems incredibly simple and I can only assume I'm doing something wrong but what?

````
         <mat-row
            *matRowDef="let row; columns: displayedColumns"
            (click)="viewUser(row)"
            class="hover:bg-blue-100 dark:hover:bg-hover hover:cursor-pointer"
         ></mat-row>
````

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by caboodal** [reply]
Hi, just setup a test project from scratch and obviously it all works perfectly so there's something wrong else where within my actual application.
